### ISSUE TITLE
Anonymous access to Mingle is disabled

### ISSUE THREAD

**Issue by dcorking** [rapidftr/tracker#154]
To reproduce (when not logged in to Mingle) :

Follow the link from Developer page, http://rapidftr.com/developer

to the Mingle instance,  https://minglehosting.thoughtworks.com/rapidftr/projects/rapidftr

and see

> This instance of Mingle is in violation of the registered license 
> ... Any anonymous access enabled projects will be inaccessible

I guess the issue is upstream at Mingle, as access for open source projects is meant to be unlimited.

I used Google Chrome Version 35.0.1916.153 on Fedora 20.

**Comment by baziwane** [comment]
Hi @dcorking, we are no longer using mingle for RapidFTR, we've migrated to using [the tracker repository on Github](https://github.com/rapidftr/tracker) for tracking upcoming stories. Please follow the instructions on the page in case you're looking to start contributing. The developer page will be updated to point to the right place in due course.

**Comment by dcorking** [comment]
Thanks, that sounds great!  Before we close this issue,I guess it would be good to update the developer instructions to point to the github tracker instead of Mingle.  Who has access?
1. http://rapidftr.com/developer (is that page on github?) and 
2. the last two bullets of the 'initial resources' post at the top of https://groups.google.com/forum/#!forum/rapidftr

**Comment by baziwane** [comment]
You're right, we're working with @scampo to have developer instructions updated, unfortunately it's not yet on Github.
